### PR TITLE
Update CODEOWNERS for static linking code

### PR DESCRIPTION
**Pull request by MaxDesiatov**
Added myself to track changes related to static linking on Linux.

### CODE REVIEW

**Comment by MaxDesiatov** [comment]
@swift-ci smoke test

**Review by tomerd — APPROVED** [review]

**Comment by MaxDesiatov** [comment]
@swift-ci smoke test and merge

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -97,6 +97,7 @@
 /lib/DependencyScan                                 @artemcm
 /lib/Driver                                         @artemcm
 /lib/DriverTool/swift_symbolgraph_extract_main.cpp  @QuietMisdreavus
+/lib/DriverTool/autolink_extract_main.cpp           @MaxDesiatov @etcwilde
```
Comment (AnthonyLatsis) [inline .github/CODEOWNERS:100]: I think this should be ordered before the previous one.

**Comment by MaxDesiatov** [comment]
@swift-ci smoke test and merge

**Comment by MaxDesiatov** [comment]
@swift-ci test windows

**Review by xedin — APPROVED** [review]

**Pull request merged**